src/index.js

```diff
@@ -0,0 +1,6 @@
+// programmatic usage
+const Linter = require('standard-engine').linter;
+
+const opts = require('./options.js');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a standard "import" statement instead of "require(...)". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS3533)